srcpkgs/sagemath/patches/numpy_2.1b.patch

```diff
@@ -0,0 +1,223 @@
+diff --git a/src/sage/combinat/fully_packed_loop.py b/src/sage/combinat/fully_packed_loop.py
+index b3ebd206cd2..d61242928dc 100644
+--- a/src/sage/combinat/fully_packed_loop.py
++++ b/src/sage/combinat/fully_packed_loop.py
+@@ -68,7 +68,7 @@ def _make_color_list(n, colors=None, color_map=None, randomize=False):
+ 
+         sage: import numpy as np
+         sage: if int(np.version.short_version[0]) > 1:
+-        ....:     np.set_printoptions(legacy="1.25")
++        ....:     np.set_printoptions(legacy="1.25") and None
```

Comment:
@ahesford this works but I'm not convinced, what do you think?

Point is,
- in numpy 2.0, `np.set_printoptions()` returns `None` which sagemath does not print.
- in numpy 2.1, it returns some object which sagemath prints. I hoped semicolon would do the trick, but it doesn't.

Reply:
Maybe this is the reasonable way:
```suggestion
         sage: if int(np.version.short_version[0]) > 1:
-        ....:     np.set_printoptions(legacy="1.25")
+        ....:     _token = np.set_printoptions(legacy="1.25")
```